alpine

`sudo docker run --interactive --rm --tty alpine:latest` then install rustup via https://rustup.rs/ then:

```
apk add gcc musl-dev pkgconf               # needed for most other Rust projects
apk add freetype-static                    # needed for "linking with `cc`"
apk add fontconfig-static fontconfig-dev   # needed for parley
apk add wayland-static wayland-dev         # needed for bevy

cargo install --git https://github.com/orhun/ratty.git  # fails!
```

```
...
          /usr/lib/gcc/x86_64-alpine-linux-musl/15.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: fcxml.c:(.text+0x2f1): undefined reference to `XML_GetCurrentLineNumber'
...
          /usr/lib/gcc/x86_64-alpine-linux-musl/15.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: sfnt.c:(.text+0x59d2): undefined reference to `png_create_info_struct'
...
          /usr/lib/gcc/x86_64-alpine-linux-musl/15.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: ftgzip.c:(.text+0x604): undefined reference to `inflateEnd'
...
          /usr/lib/gcc/x86_64-alpine-linux-musl/15.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: ftbzip2.c:(.text+0x235): undefined reference to `BZ2_bzDecompressInit'
...
```

okay, i have to go, but alpine still needs work to get it to finish `cargo build --release` (and it's probably lower in priority than other desktop Linux distributions)

--- a/README.md
+++ b/README.md
@@ -38,7 +38,6 @@ Ever wondered what's _behind_ the terminal? Press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>
 
 Requirements:
 
-- Rust toolchain with Cargo
 - A GPU / graphics stack supported by Bevy and wgpu
 - Melted cheese (optional but recommended)
 
@@ -59,6 +58,13 @@ pacman -S ratty
 Prebuilt binaries are available on the [GitHub releases page](https://github.com/orhun/ratty/releases) for direct download.
 
 ### From Git
+
+Requirements:
+
+- Rust toolchain with Cargo
+- on Bazzite / Bluefin: `rpm-ostree install gcc fontconfig-devel wayland-devel` (then reboot)
+- on Debian / Ubuntu: `apt-get update ; apt-get install gcc pkgconf libfontconfig-dev libwayland-dev`
+- on Fedora: `dnf install gcc fontconfig-devel wayland-devel`
 
 ```bash
 cargo install --git https://github.com/orhun/ratty

